device/dhcp/common/dhcp_proposal.md

```diff
@@ -0,0 +1,703 @@
+1. Array of dhcp clients over a eth not a valid usecase or not. OC model has only reference to dhcpclient [previous comment taken care]: However as per https://www.ietf.org/rfc/rfc2131.txt 4.2 DHCP server administrative controls (its possible to have multiple dhcp-clients with different client id over same mac)
```

Comment:
Is this .md file kept intentionally in the model ? Probably need to remove.

Reply:
we will removed